src/Schema/Events/index.ts

```diff
@@ -501,6 +503,7 @@ export type Event =
   | TappedRequestPriceEstimate
   | TappedSell
   | TappedSellArtwork
+  | TappedShare
```

Comment:
@louislecluse I forgot to add this when I worked on https://github.com/artsy/cohesion/pull/579 for tappedShare. Is this necessary?